src/elfpy/pricing_models/yieldspace.py

```diff
@@ -303,7 +303,7 @@ def calc_in_given_out(
             \begin{cases}
             c (\frac{1}{\mu} (\frac{k - (2y + cz - \Delta y)^{1-t}}{\frac{c}{\mu}})^{\frac{1}{1-t}} - z),
```

Comment:
While we're fixing the use of time in comments, let's do it everywhere. For example this equation and any others you see that still use `t` instead of `\tau`.